test/dag-node-test.js

```diff
@@ -29,10 +29,10 @@ module.exports = (repo) => {
         expect(Buffer.isBuffer(node.data)).to.be.true.mark()
         expect(node.size).to.be.above(0).mark()
 
-        util.serialize(node, (err, serialized) => {
+        DAGNode._serialize(node, (err, serialized) => {
```

Comment:
This doesn't make sense. Why would the serialization functionality be separate from all other functionality of a specific data structure (DAGNode)?

Also, not possible atm as deserialize uses .create() (two-way coupling between utils and dag-node-utils).